src/operator/tensor/elemwise_unary_op.cc

```diff
@@ -90,7 +93,7 @@ For example::
   })
 .set_attr<FCompute>("FCompute<cpu>", CastCompute<cpu>)
 .set_attr<nnvm::FGradient>("FGradient", ElemwiseGradUseNone{"_backward_cast"})
-.add_argument("data", "NDArray-or-Symbol", "Source input")
+.add_argument("data", "ndarray-or-symbol", "The input.")
```

Comment:
Change to "NDArray-or-Symbol" <- should be capitalized to be consistent with convention in Python docs.